packages/custom_lint_core/lib/src/configs.dart

```diff
@@ -35,13 +39,28 @@ class CustomLintConfigs {
     final include = yaml['include'] as Object?;
     var includedOptions = CustomLintConfigs.empty;
     if (include is String) {
-      final includeAbsolutePath = absolute(
-        analysisOptionsFile.parent.path,
-        include,
-      );
-      includedOptions = CustomLintConfigs.parse(
-        analysisOptionsFile.provider.getFile(includeAbsolutePath),
-      );
+      final includeUri = Uri.parse(include);
+      String? includeAbsolutePath;
+
+      if (include.startsWith('package:')) {
```

Comment:
You already parsed an Uri before. We can use that to get the scheme:

```suggestion
      if (includeUri.scheme == 'package') {
```